lib/publiccloud/utils.pm

```diff
@@ -133,6 +134,19 @@ sub register_openstack {
     $instance->ssh_assert_script_run(cmd => $cmd, timeout => 700, retry => 5);
 }
 
+# Validation for update repos
+sub validate_repo {
+    my ($maintrepo) = @_;
+    my ($incident, $type) = ($2, $1) if ($maintrepo =~ /\/(PTF|Maintenance):\/(\d+)/g);
```

Comment:
ofc. it is less likely but if we want to handle unexpected situations let's also take into account when string didn't match in such a case no point to print undef

Reply:
ok now I see that you just moved code from old place to a function , feel free to ignore my comment as in a fact you haven't introduced this code . Nevertheless if you will tweak it IMHO it will be nice improvement

Reply:
Thank you for the proposal, implemented!